### DISCUSSION TITLE
参考文献不显示

### DISCUSSION THREAD

**Discussion by pkeke** [note286/xduts#25] [Q&A] [answered]
大佬你好，使用了你那个论文模板，但是不知道为什么参考文献显示不出来。
配置如下：
![1](https://user-images.githubusercontent.com/44956389/208691142-b40e050f-20d1-4ea1-9128-4fb4a1b15ab2.png)
tex.bib文件如下：
![3](https://user-images.githubusercontent.com/44956389/208692288-69a0059f-de7c-4b0a-870b-cb6e16673cff.png)
打印的日志如下：
![2](https://user-images.githubusercontent.com/44956389/208691152-3c127b42-bec7-4a94-a588-38c27afbe54c.png)

**Comment by note286** [comment, marked as answer]
- 代码请使用代码块环境以文本的形式给出，不要以图片的形式给出，请节约对方时间。

- 手册中已经提及参考文献使用`biber`或者`bibtex`编译。

- 既然你知道看日志，你是否忽视了你所述日志下方有如下内容（注意倒数第三行）：

  ```
  第一章
  
  LaTeX Warning: Citation '1967' on page 1 undefined on input line 18.
  
  [1] [2]
  
  LaTeX Warning: Empty bibliography on input line 23.
  
  [3] [4] [5] [6] [7] (./1.aux)
  
  LaTeX Warning: There were undefined references.
  
  
  Package rerunfilecheck Warning: File `1.out' has changed.
  (rerunfilecheck)                Rerun to get outlines right
  (rerunfilecheck)                or use package `bookmark'.
  
  
  Package biblatex Warning: Please (re)run Biber on the file:
  (biblatex)                1
  (biblatex)                and rerun LaTeX afterwards.
  ```

- 以上结果的测试代码如下所示，编译命令为`xelatex-biber-xelatex`：

  ```latex
  \documentclass{xdupgthesis}
  \xdusetup{info={bib-resource={./bib/tex.bib}}}
  \begin{document}
  \chapter{title}
  \cite{1967}
  \end{document}
  ```

**Reply by pkeke** [reply]
非常感谢你的解答，是我自己的疏忽，没有看下面的日志，问题已解决！
